exporter/loadbalancingexporter/resolver_k8s.go

```diff
@@ -0,0 +1,301 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package loadbalancingexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/loadbalancingexporter"
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net"
+	"sort"
+	"strings"
+	"sync"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/tag"
+	"go.uber.org/zap"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/clientcmd"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+var _ resolver = (*k8sResolver)(nil)
+
+var (
+	errNoSvc                        = errors.New("no service specified to resolve the backends")
+	errNoSvcPorts                   = errors.New("no ports specified to resolve the backends")
+	k8sResolverMutator              = tag.Upsert(tag.MustNewKey("resolver"), "k8s service")
+	k8sResolverSuccessTrueMutators  = []tag.Mutator{resolverMutator, successTrueMutator}
+	k8sResolverSuccessFalseMutators = []tag.Mutator{resolverMutator, successFalseMutator}
+)
+
+type k8sResolver struct {
+	logger    *zap.Logger
+	service   string
```

Comment:
Perhaps something like: 

```suggestion
	svcName string
    svcNs   string
```

Reply:
done